lms/djangoapps/oauth_dispatch/adapters/dot.py

```diff
@@ -12,7 +12,8 @@ class DOTAdapter(object):
 
     backend = object()
 
-    def create_confidential_client(self, name, user, redirect_uri, client_id=None):
+    def create_confidential_client(self, name, user, redirect_uri, client_id=None,
+                                   authorization_grant_type=models.Application.GRANT_AUTHORIZATION_CODE):
```

Comment:
Nit: per our styling guidelines, arguments should either all be on one line, or on separate lines, but not mixed.